soccer/gameplay/standard_play.py

```diff
@@ -0,0 +1,46 @@
+import play
+import ui
+import tactics
+
+
+## @brief A standardized play that handles actions that an average play needs
+# Right now, this is only used to implement a standard way to run or not run
+# the play with defense, but any action that a normal play should do can be 
+# placed here
+class StandardPlay(play.Play):
+
+    #Performs actions that all "Standard Plays" should do on initialization
+    #Note: This method is called many times during the duration of a play,
+    #Not just on selection
+    def __init__(self, continuous):
+        super().__init__(continuous)
+        self.use_standard_defense()
+
+        #If the "Use Defense" checkbox is checked and the play isn't already running
```

Comment:
It looks a little odd that this is the only comment indented away from the method's indentation